### DISCUSSION TITLE
無人機、大數據，烏克蘭與俄羅斯打響 AI 戰爭第一槍？

### DISCUSSION THREAD

**Discussion by ruthchen1020** [1101-GS4719-C/Python-Community-GS4719-C#889] [General]
我覺得AI未來不僅在作戰計畫上有很大的進步，AI武器也是人類需要更加注意的點。未來的無人機、槍枝會不會也裝上AI自動瞄準射擊系統，殺人已經不需要一個”人類”去執行，這樣的AI機器人取得的勝利是人類所期望的嗎，而人類的生命又應該如何被看待。
https://buzzorange.com/techorange/2022/03/01/russia-ukraine-ai-war/

**Comment by StevenWanglolz** [comment]
AI是人類科技的一大突破，真的很不希望它被利用在戰爭之中，造成更多的傷亡。科技的進步應該是要造福人類，而不是讓我們自相殘殺!

**Comment by Chiway1229** [comment]
雖然科技一方面改善了我們的生活，但也能當成傷害人的武器
萬物都是一種雙刃劍呢

**Comment by bonny123865** [comment]
AI的確能應用在好跟壞的方面，對於AI武器的研發
感覺如果未來世界發生大規模戰爭，會造成更多人員傷亡。
科技進步不僅帶來便利，但若是不當運用，也會造成傷害。

**Comment by Kotori0501** [comment]
真的不太希望能將AI應用在戰爭中，因為人的生命是有限的。
假如可利用無人機進行遠端操控，感覺死亡人數會變得更多。

**Comment by evanchou1204** [comment]
一旦士兵過度依賴AI瞄準系統，是否也代表著失去了AI就失去了戰鬥力，那攻擊敵方的AI系統就成為未來的戰爭核心了。

**Comment by ching020721** [comment]
AI本身可能是可以帶領人類一起進步的，但若被人類濫用可能會導致過壞的結果

**Comment by Evan0979** [comment]
之前冷戰的時候有太空競賽,下一波會不會是AI競賽咧

**Comment by CPCpyt** [comment]
在技術對等的情況，作戰還是考驗決策以及行動能力。
我想這樣的技術還無法完全翻轉戰爭的邏輯和型態。

**Comment by swerfg03joe** [comment]
我認為AI這個詞好像有點過於氾濫了，現代戰爭本來就是打資訊戰，與其蒐集一堆間接資訊再透過AI的數學模型去分析(搞不好還是錯的)還不如用個解析度高義點的衛星，或是大量使用搭載攝影設備的小型無人機飛過去看還比較實在，畢竟體型小被發現也很難打下來

**Comment by s96015welson** [comment]
常常聽信號與系統老師提及只靠台灣自己做不出來一台無人機更何況可以載炸彈的無人機，這才意識到現代戰爭形勢已經與過往不同了，更強調科技，而台灣也需要在這個領域多加油了。

**Comment by HsuChen88** [comment]
所有國家都在做AI，但如今AI若成為互相傷害的武器，人的存在甚至比AI還不值。

**Comment by 110302550** [comment]
我覺得AI讓我心裡都充滿恐懼，因為她是未知數，不知道他在想甚麼，隨時可能產生出自己的意識我們也來不及應對，假如他們之後自動發起戰爭我們又該怎麼辦呢?

**Comment by ghost** [comment]
AI武器能以最小的成本產生最大殺傷，然對於被攻擊的人來說，這彷彿是用到外掛般，不僅開透視還打的準。只願世界沒有戰爭
